conf/reflect-config.json

```diff
@@ -3271,6 +3271,12 @@
   "allDeclaredMethods":true,
   "methods":[{"name":"fromValue","parameterTypes":["java.lang.String"] }]
 },
+{
+  "name":"io.seqera.tower.model.GoogleBatchConfig",
```

Comment:
Instead of creating a completely new item we should instead add the `getNextflowConfig` method entry to the already existing `GoogleBatchConfig` definition (line 2709)
https://github.com/seqeralabs/tower-cli/blob/22137b9fb44b1fcd70baaa898ec9dd05df79c582/conf/reflect-config.json#L2709

Reply:
done